### PR TITLE
fix(edit): requires GHES >= 3.2

### PR DESCRIPTION
**Pull request by sileht**
Change-Id: Iac2936b7140c19e9f9bd750b751a57d064c5cf06

### CODE REVIEW

File: mergify_engine/tests/functional/actions/test_edit.py [hunk 1]
```diff
@@ -14,12 +14,18 @@
...
 
+from mergify_engine import config
 from mergify_engine.tests.functional import base
 
 
 class TestEditAction(base.FunctionalTestBase):
+    @pytest.mark.skipif(
+        not config.GITHUB_URL.startswith("https://github.com"),
+        reason="requires GHES 3.2",
+    )
```
Comment (jd) [inline mergify_engine/tests/functional/actions/test_edit.py:28]: This means we won't test it against GHES >= 3.2 either? 😢

Comment (sileht) [reply mergify_engine/tests/functional/actions/test_edit.py:28]: For now yes, the integration run 3.1 so we are OKish, I created a ticket to retrieve the server version and be able to write a better skipif.

Comment (Syffe) [reply mergify_engine/tests/functional/actions/test_edit.py:28]: would it be more understandable with `requires GHES==3.2` ? Otherwise LGTM

Comment (jd) [reply mergify_engine/tests/functional/actions/test_edit.py:28]: It's actually >= 3.2

**Pull request merged**